.github/scripts/generate_docs.py

```
@@ -308,79 +394,121 @@ def prepare_pandoc_input(file_path: Path, literate_c_script: Path) -> str:
 
 
 def run_pandoc(pandoc_input: str, output_html_path: Path, template_path: Path, 
-               base_url: str, wiki_title: str, page_url: str, page_title: str) -> str:
+               base_url: str, wiki_title: str, page_url: str, page_title: str,
+               seo_metadata: Dict[str, str] = None) -> str:
+    """
+               """Converts Markdown content to a standalone HTML document using Pandoc.
```

Ruff (0.8.2)

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Expected 'in', found name

---

SyntaxError: Expected ':', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found 'and'

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an expression

---

SyntaxError: Expected ',', found name

---

SyntaxError: missing closing quote in string literal

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an expression

---

SyntaxError: Expected an expression

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Expected ',', found newline

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected 'else', found name

---

SyntaxError: missing closing quote in string literal

---

SyntaxError: Expected ',', found newline

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Expected 'in', found name

---

SyntaxError: Expected ':', found name

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an identifier

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected an identifier

---

SyntaxError: Expected an identifier

---

SyntaxError: Expected ':', found dedent

---

SyntaxError: Expected an expression

---

SyntaxError: Unexpected indentation

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected 'import', found 'if'

---

SyntaxError: Expected ':', found name

---

Multiple statements on one line (semicolon)

(E702)